portfolio/src/main/webapp/images/logo.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg width="100%" height="100%" viewBox="0 0 2732 2048" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
```

Comment:
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

> While there is no column limit recommendation for HTML, you may consider wrapping long lines if it significantly improves readability.

Reply:
But it's auto generated, I think. I uploaded this SVG file as image. Should I change it?

Reply:
If it's autogenerated it's ok. Optional: Mark it as a binary file in git. (e.g. https://www.programming-books.io/essential/git/identify-binary-files-cf8308a172e34112bfd725d57ab95f4d)